templates/flower/flower-deployment.yaml

```diff
@@ -48,6 +48,19 @@ spec:
         - name: {{ template "registry_secret" . }}
       {{- end }}
       containers:
+        {{- if (and .Values.authSidecar.enabled ( eq .Values.executor "CeleryExecutor" )) }}
```

Comment:
```suggestion
        {{- if .Values.authSidecar.enabled }}
```
This whole doc is already only shown for CeleryExecutor so you dont need to duplicate the check here.

Reply:
nice catch @pgvishnuram ^

Reply:
changes applied